diff --git a/node_modules/prettier/third-party.js b/node_modules/prettier/third-party.js
index 5f1ac8e..b22d6cc 100644
--- a/node_modules/prettier/third-party.js
+++ b/node_modules/prettier/third-party.js
@@ -9660,7 +9660,7 @@ var readFile_1 = createCommonjsModule(function (module, exports) {
       const content = await fsReadFileAsync(filepath, 'utf8');
       return content;
     } catch (error) {
-      if (throwNotFound === false && error.code === 'ENOENT') {
+      if (throwNotFound === false && error.code === 'ENOENT' || error.code === "ENOTDIR") {
         return null;
       }
 
@@ -9676,7 +9676,7 @@ var readFile_1 = createCommonjsModule(function (module, exports) {
 
       return content;
     } catch (error) {
-      if (throwNotFound === false && error.code === 'ENOENT') {
+      if (throwNotFound === false && error.code === 'ENOENT' || error.code === "ENOTDIR") {
         return null;
       }
 
@@ -9747,7 +9747,7 @@ function isTypeSync(fsStatType, statsMethodName, filePath) {
   try {
     return fs__default['default'][fsStatType](filePath)[statsMethodName]();
   } catch (error) {
-    if (error.code === 'ENOENT') {
+    if (error.code === 'ENOENT' || error.code === "ENOTDIR") {
       return false;
     }
 
